### DISCUSSION TITLE
Wrong trnslation error in polish version

### DISCUSSION THREAD

**Discussion by karaspr** [open-meteo/open-meteo#1085] [General]
There is a mistake in translation into polish. English word visibility is translated as "widoczność" when correct word is "widzialność".

**Comment by patrick-zippenfenig** [comment]
Hello, Open-Meteo is only available in English and doesn't offer translations. Could this translation be part of a third-party app?

**Comment by karaspr** [comment]
Perhaps. I will try to find out more. Thanks.


Przemysław Karaś

pon., 28 paź 2024 o 08:48 Patrick Zippenfenig ***@***.***>
napisał(a):
